test-integration/scripts/20-no-memory-limit-elasticsearch.sh

```diff
@@ -5,7 +5,7 @@ source $(dirname $0)/00-init-env.sh
 
 cd "$JHI_FOLDER_APP"
 if [ -a src/main/docker/elasticsearch.yml ]; then
-    sed -i -e 's/        environment:/        # environment:/1;' src/main/docker/elasticsearch.yml
+    #sed -i -e 's/        environment:/        # environment:/1;' src/main/docker/elasticsearch.yml
```

Comment:
as this line will be useless, could you remove it, instead of comment it ?

Reply:
Done